scripts/testnet/include/config.sh

```diff
@@ -89,7 +89,12 @@ updateSeednodeConfig() {
 prepareElasticsearch() {
   echo "Starting Elasticsearch Docker container..."
   pwd
-  export ES_CONTAINER_ID=$(docker run -d -p 9200:9200 -p 9301:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:7.10.2)
+  ES_CONTAINER="sov-elastic"
```

Comment:
Is it mandatory to add `sov` prefix? 
I think this should work for normal chain as well

Reply:
renamed to `mvx-elastic` to be more unique